Jenkinsfile

```
@@ -84,3 +83,32 @@ demos['databound'] = {
 }
 
 parallel(demos)
+
+def branchName = currentBuild.projectName
+if (!branchName.startsWith('PR-')) {
+    node('docker') {
+        def image
+        def imageName = 'jenkins/jenkinsfile-runner-experimental'
+        def imageTag
+
+        stage('Build container') {
+            timestamps {
+                deleteDir()
+                def scmVars = checkout scm
+
+                def shortCommit = scmVars.GIT_COMMIT
+                imageTag = "${env.BUILD_ID}-build${shortCommit}"
```

It will lead to a huge number of tags in DockerHub.
Could we start from ...

```suggestion
                imageTag = branchName.equals("master") ? "latest" : branchName
```

It would produce images for all branches in the main repo, but it will not spam new tags for each commit